server/sonar-server/src/main/java/org/sonar/server/es/IndexingListener.java

```diff
@@ -19,13 +19,23 @@
  */
 package org.sonar.server.es;
 
-import java.util.Collection;
+import java.util.List;
 
 public interface IndexingListener {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1214)